package.json

```diff
@@ -33,7 +33,7 @@
     "build:es": "yarn lerna exec --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments'",
     "build:ts": "yarn lerna exec --stream --sort 'tsc -b tsconfig.build.json'",
     "clean": "rm -Rf packages/*/lib",
-    "prepublishOnly": "yarn test",
+    "prepublishOnly": "yarn jest packages/sdk/",
```

Comment:
How come we are limiting the prepublish tests to just the sdk?

Reply:
Because everything else has to work correctly for the TS SDK to pass these tests and be deployable. We can extend it to all sdk packages by using `/sdk` instead of `/sdk/` due to pattern matching but I don't think we need to go that wide yet. All our other packages aren't currently being published. When api-explorer is ready to be published, then we can widen the scope.

Reply:
Makes sense